ubuntu_bridge_network_configuration.sh

```diff
@@ -10,16 +10,23 @@ source lib/common.sh
 if [ "$MANAGE_PRO_BRIDGE" == "y" ]; then
      # Adding an IP address in the libvirt definition for this network results in
      # dnsmasq being run, we don't want that as we have our own dnsmasq, so set
-     # the IP address here
+     # the IP address here.
+     # Create a veth iterface peer.
+     sudo ip link add "$PRO_IF1" type veth peer name "$PRO_IF2"
```

Comment:
what if the user sets PRO_IF1 to be a physical interface of his machine ? and defines PRO_IF2 to "" ?

Reply:
Fixed based on our discussion.